package.json

```diff
@@ -18,7 +18,7 @@
     "build": "yarn run lerna run build",
     "report-coverage": "yarn run lerna run test-coverage",
     "test:watch": "node node_modules/jest/bin/jest.js --watch",
-    "postinstall": "yarn run lerna run postinstall && patch-package && sh scripts/custom_patch_packages.sh && yarn keys:decrypt",
+    "postinstall": "yarn run lerna run postinstall && patch-package && yarn keys:decrypt",
```

Comment:
This means we can stop "copying" scripts folder inside docker files!

Reply:
@jeanregisser Looks like your patch fix had an unexpected benefit